scripts/multi-acc-new-eks-mixed-observability-pattern/amg-preconfig.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
instead of `scripts` rename the folder as `helpers`

Reply:
renamed